x/marker/keeper/marker.go

```diff
@@ -271,6 +283,9 @@ func (k Keeper) BurnCoin(ctx sdk.Context, caller sdk.AccAddress, coin sdk.Coin)
 		if err = m.SetSupply(total); err != nil {
 			return err
 		}
+		if err := m.Validate(); err != nil {
+			return err
```

Comment:
linter seems to be legit problem here?

Reply:
Yeah, I saw that and fixed it.  Forgot to run lint before I checked it in...